spec/support/helpers.rb

```diff
@@ -6,3 +6,12 @@ def user_login
   fill_in 'Password', with: user.password
   click_on 'Log in'
 end
+
+def admin_login
+  user = FactoryGirl.create(:user, role: "admin")
+  visit root_path
+  click_on 'Login'
+  fill_in 'Email', with: user.email
+  fill_in 'Password', with: user.password
+  click_on 'Log in'
+end
```

Comment:
While this is perfectly legit, it might be beneficial to have a method where you can pass in a user to log them in. This will eliminate some duplication:

```
def log_in_user(user)
...
```